### ISSUE TITLE
User defined layer in MatConvNet

### ISSUE THREAD

**Issue by Addhi86** [vlfeat/matconvnet#815]
Hello, I have recently started using MatConvNet. I want to know that is it possible to add user defined layer in MatConvNet?

**Comment by bazilas** [comment]
Yes. It might more convenient to do it using the dagnn structure.
